src/main/java/org/cbioportal/cancerstudy/CancerStudyMetadata.java

```diff
@@ -0,0 +1,24 @@
+package org.cbioportal.cancerstudy;
+
+import java.util.Date;
+
```

Comment:
@haynescd hey, i came across this beauty.   this is to provide default null values for some of these props?